bluesky/plans.py

```diff
@@ -334,8 +334,8 @@ def rel_list_grid_scan(detectors, *args, per_step=None, md=None):
     @bpp.reset_positions_decorator(motors)
     @bpp.relative_set_decorator(motors)
     def inner_relative_list_grid_scan():
-        return (yield from list_grid_scan(detectors, *args, per_step=per_step,
-                                          md=_md))
+        return (yield from list_grid_scan(detectors, *args, snake_axes,
```

Comment:
Any parameters after `*args` must be passed by keyword (as you'll discovered when you write a test for this).

Reply:
ok thanks, this will be fixed in the next commit !-)